Formula/libgr.rb

```diff
@@ -0,0 +1,48 @@
+class Libgr < Formula
+  desc "GR framework: a graphics library for visualisation applications"
+  homepage "https://gr-framework.org/"
+  url "https://github.com/sciapp/gr/archive/v0.37.0.tar.gz"
+  sha256 "9120c4215cd2c10239f45f4cbf79dcb199a9e7e3bbbf36e5ce8f4e5eb2e70e24"
+  depends_on :xcode => :build
+  depends_on "cairo"
+  depends_on :x11
+
+  def install
+    args = %W[
+      GRDIR=#{prefix}
+    ]
+
+    system "make", "self", *args
+    system "make", "clean"
+  end
+
+  test do
+    (testpath/"test.c").write <<~EOS
+      #include <stdio.h>
+      #include <gr.h>
+
+      int main(void) {
+          gr_beginprint("test.png");
+          double x[] = {0, 0.2, 0.4, 0.6, 0.8, 1.0};
+          double y[] = {0.3, 0.5, 0.4, 0.2, 0.6, 0.7};
+          gr_polyline(6, x, y);
+          gr_axes(gr_tick(0, 1), gr_tick(0, 1), 0, 0, 1, 1, -0.01);
+          gr_endprint();
+          return 0;
+      }
+    EOS
+
+    flags = %W[
+      -I#{include}
+      -L#{lib}
+      -lGR
+    ]
+
+    system ENV.cc, "test.c", "-o", "test", *flags
+
+    ENV["GKS_WSTYPE"] = "5"
```

Comment:
What's that?

Reply:
This is a workaround to run it headless. I've asked on the issue tracker for a different solution: https://github.com/sciapp/gr/issues/81

Reply:
removed